src/sentry/api/endpoints/chunk.py

```diff
@@ -31,6 +31,7 @@
     "bcsymbolmaps",  # BCSymbolMaps and associated PLists/UuidMaps
     "il2cpp",  # Il2cpp LineMappingJson files
     "portablepdbs",  # Portable PDB debug file
+    "dartsymbols",  # JSON file containing obfuscated and deobfuscated Dart symbols
```

Comment:
There isn't really an established name for this kind of file. I just named it `dartsymbols` here to find something quick just to validate this is how we would add the file and then I was going to make the `sentry-cli` changes.

Feel free to let me know if we have some kind of better naming convention